src/redux/requests/siteRequests.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+export function callSite() {
+  const config = {
+    headers: { 'Content-Type': 'application/json' },
+    withCredentials: true,
+  };
+  return axios.get('/api/site', config)
+    .then(response => response.data)
+    .catch((error) => { 
+        throw error.response || error; 
+    });
+}
+
+export function callSetSite(id) {
```

Comment:
Please add a comment that explains exactly what this function is doing. Also, can you format the spacing so it's easier to understand?

Reply:
Also, please check the documentation for the route you're using. You need to give it exactly what it wants. Check how to write routes with params (the flower garden from week 8 has an example in the delete).